qemu/tests/cfg/pvpanic_memory_leak.cfg

```diff
@@ -0,0 +1,12 @@
+- pvpanic_memory_leak:
+    virt_test_type = qemu
```

Comment:
We can add a limitation to limit the guest type because we only need to test this case with Windows guest.
`only windows`